drive/client.bal

```diff
@@ -208,7 +208,7 @@ public client class Client {
     #             'modifiedTime', 'name', 'name_natural', 'quotaBytesUsed', 'recency', 'sharedWithMeTime', 'starred', 
     #              and 'viewedByMeTime'
     # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
-    @display {label: "Get forms by name"}
+    @display {label: "Get Forms By Name"}
```

Comment:
Shall we say `Get Forms` ? The input parameter `Name` is obvious when he loads the input form.

Reply:
Hi, 
Updated in all the places.
Thanks